docs/source/using_gridappsd/rc1_overview.rst

```diff
@@ -51,7 +77,7 @@ For an orderly shutdown of the demonstration:
 
 |rc1_overview_image0|
 
```

Comment:
Should these images be changed to reflect the rc3

Reply:
Created a new pull request with updated images and Future Enhancements removed.

https://github.com/GRIDAPPSD/gridappsd-docs/pull/1